buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -0,0 +1,210 @@
+@file:Suppress("RedundantVisibilityModifier")
+
+package io.getstream.chat.android
```

Comment:
Moved to the proper package just for consistency. Also grouped versions inside a `Versions` object so that they can be reasonably referenced in Gradle scripts when needed. Only exposed Kotlin and Compose versions for now.